docs/scos/dev/migration-program/migration-to-paas/paas-migration-documents/upgrade-project-packages.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Upgrade project packages
```

Comment:
How are these packages different from the `all used packaged` that we are updating in the prev step?